www/grpc/server_test.go

```diff
@@ -114,9 +114,7 @@ func (td *testData) blockchainClient(t *testing.T) (*grpc.ClientConn, pactus.Blo
 	conn, err := grpc.NewClient("passthrough://bufnet",
 		grpc.WithContextDialer(td.bufDialer),
 		grpc.WithTransportCredentials(insecure.NewCredentials()))
-	if err != nil {
-		t.Fatalf("Failed to dial blockchain server: %v", err)
-	}
+	assert.NoError(t, err)
```

Comment:
nit: here (and in other test helper functions) we could use `require.NoError(t, err)` in order to stop test execution earlier if error is found

Reply:
Yes, its already done. some of tests was missing it and I've added them on this PR.